recipes/fs-access-mode-constants/src/workflow.ts

```diff
@@ -0,0 +1,283 @@
+import type { Edit, SgRoot } from "@codemod.com/jssg-types/main";
+import { getNodeRequireCalls } from "@nodejs/codemod-utils/ast-grep/require-call";
```

Comment:
```suggestion
port type Js from "@codemod.com/jssg-types/langs/javascript";
import { getNodeRequireCalls } from "@nodejs/codemod-utils/ast-grep/require-call";
```

You can import the grammar to have a better typecking in query and usage is `SgNode<Js>` or `SgRoot<Js>`